internal/prometheus_exporter/server.go

```diff
@@ -17,47 +17,56 @@ var (
 
 func newLimitsCollector() *LimitsCollector {
 	return &LimitsCollector{
-		LimitTotal: prometheus.NewDesc(prometheus.BuildFQName("github", "limit", "total"),
+		LimitTotal: prometheus.NewDesc(prometheus.BuildFQName("github", "rate_limit", "total"),
```

Comment:
I agree with `rate_limit` being a better name, but this change is an upstream breaking change, as all existing users will have their metrics renamed if we do this. Perhaps we should continue to use `limit` 🤔

Reply:
Hadn't thought of it 🤔 the original repo is versioned, so users could still hang onto that, but it is indeed breaking, and probably not worth the trouble for a _slightly_ better name. 

I was full on thinking about our own case and not about taking it back to the original, because I'm still gonna try to make some more changes to this repo to allow for metrics for multiple github accounts. Right now it's limited to one. And that's gonna be a major change in how the image is run...

Still, may make sense to change this one back and present it as an offering to $originalAuthor :sisi: I'll change it back